glean/metrics.yaml

```
@@ -120,20 +114,58 @@ page:
       notification_emails:
         - marketing-websites-team@mozilla.com
       expires: never
-  page_event:
+  page_hit:
```

```suggestion
  hit:
```

`page` is already given by the category on line 4: if you add it again here your event will end up named `pagePageHit` instead of  `pageHit`